scipy/stats/_distn_infrastructure.py

```diff
@@ -2322,37 +2322,6 @@ def fun(x, *args):
 ## Handlers for generic case where xk and pk are given
```

Comment:
The two left-over ``_drv2`` methods are for ``values=None``, so this comment is incorrect. Can you delete it?